ParseStarterProject/watchOS/ParseStarterProject-Swift/ParseStarterProject/AppDelegate.swift

```diff
@@ -39,42 +39,42 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
             // Enable storing and querying data from Local Datastore.
             // Remove this line if you don't want to use Local Datastore features or want to use cachePolicy.
-            $0.localDatastoreEnabled = true
+            $0.isLocalDatastoreEnabled = true
         }
-        Parse.initializeWithConfiguration(configuration)
+        Parse.initialize(with: configuration)
 
         PFUser.enableAutomaticUser()
 
         let defaultACL = PFACL()
 
         // If you would like all objects to be private by default, remove this line.
-        defaultACL.publicReadAccess = true
+        defaultACL.hasPublicReadAccess = true
 
-        PFACL.setDefaultACL(defaultACL, withAccessForCurrentUser: true)
+        PFACL.setDefault(defaultACL, withAccessForCurrentUser: true)
```

Comment:
Thank you @mtrezza  for your comment. The docs don't contain this naming. I can prepare PR to change this naming in docs. (Because it is a separate repository)

Reply:
This isn’t a name change or breaking change. Anyone using this SDK in Swift projects has been made this change as the macro in the file has been there since at least 2018: https://github.com/parse-community/Parse-SDK-iOS-OSX/blob/2e4242c683e645a7d78ff37dd34398119178c0c5/Parse/Parse/PFACL.h#L50

The same goes for many of the comments like this.